variables.tf

```
@@ -1,25 +1,30 @@
 variable "awp_cloud_account_id" {
-    description = "CLOUDGUARD_ACCOUNT_ID or EXTERNAL_AWS_ACCOUNT_NUMBER"
-    type        = string
+  description = "CLOUDGUARD_ACCOUNT_ID or EXTERNAL_AWS_ACCOUNT_NUMBER"
+  type        = string
+}
+
+variable "awp_centralized_cloud_account_id" {
+  description = "centralized account EXTERNAL_AWS_ACCOUNT_NUMBER or EXTERNAL_AWS_ACCOUNT_NUMBER"
+  type        = string
+  default     = null
 }
 
 variable "awp_scan_mode" {
-    description = "AWP scan mode <inAccount|saas>" # the valid values are "inAccount" and "saas" when onboarding the AWS account to Dome9 AWP.
-    type        = string
-    default     = "inAccount"
-    
+  description = "AWP scan mode <inAccount|saas|inAccountHub|inAccountSub>" # the valid values are "inAccount" and "saas" when onboarding the AWS account to Dome9 AWP.
```

hub sub are also valid
```suggestion
  description = "AWP scan mode <inAccount|saas|inAccountHub|inAccountSub>" # the valid values are "inAccount", "saas" "inAccountHub" and "inAccountSub" when onboarding the AWS account to Dome9 AWP.
```

---

updated thanks